inst/templates/wercker.yml

```diff
@@ -0,0 +1,5 @@
+box: rocker/hadleyverse
```

Comment:
Do you really want to use a box this large? Wercker saves the box after each build, for hadleyverse this contributes to 1:30 minutes of build time. I've played with creating a smaller base box tailored to wercker: https://github.com/krlmlr/r-wercker-base; the box is based on drd and includes devtools, lintr and covr.